packages/playwright-ct-svelte/index.d.ts

```diff
@@ -30,7 +30,7 @@ export interface MountOptions<HooksConfig extends JsonObject, Component extends
   hooksConfig?: HooksConfig;
 }
 
-interface MountResult<Component extends SvelteComponent> extends Locator {
+export interface MountResult<Component extends SvelteComponent> extends Locator {
```

Comment:
Sure! Are you referring to a test that examines how `MountResult` is utilized? _(not sure what ttest is BTW)_

example:

```typescript
// tests/components/ct-svelte/tests/fixture.spec.ts
import DefaultSlot from "@/components/DefaultSlot.svelte";
import { test as base, expect, type MountResult } from "@playwright/experimental-ct-svelte";

const test = base.extend<{ component: MountResult<DefaultSlot> }>({ // <-- is how one would use MountResult
  component: async ({ mount }, use) => {
    await use(await mount(DefaultSlot));
  },
});

test("custom mount fixture", async ({ component }) => {
  await expect(component).toContainText("Welcome!");
});
```

Reply:
Yup, search for `runTSC` or maybe we want ´utils/generate_types/test/test.ts´ for component-testing. Just an idea, probably also fine to just merge it for now, just an idea how we can prevent breaking it in the future.

Reply:
Hi @mxschmitt i'll revisit the ttest later on. I think this would be useful. I also suggested a similar approach some time ago; https://github.com/microsoft/playwright/pull/17070